lib/mapnik_backend.js

```diff
@@ -290,21 +260,65 @@ MapnikSource.prototype._populateInfo = function(callback) {
             (info.bounds[2] - info.bounds[0]) / 2 + info.bounds[0],
             (info.bounds[3] - info.bounds[1]) / 2 + info.bounds[1],
             2 ];
-
-        source._info = info;
-        source._pool.release(map);
-        callback(null)
+        this._info = info;
+        this._pool.release(map);
+        return callback(null);
     });
 };
 
+exports['calculateMetatile'] = calculateMetatile;
```

Comment:
Once you moved this function to `mapnik_nackend`, do we need to export it? I guess for testing porpuses, doesn't it?

Reply:
Yes, just for testing.